device_m1000.cpp

```diff
@@ -46,9 +46,15 @@ static const sl_channel_info m1000_channel_info[2] = {
 	{CHANNEL_SMU, "B", 3, 2},
 };
 
+static const sl_mode_info m1000_mode_info[3] = {
```

Comment:
it seems quite possible non-smu devices will be supported in pixelpulse2, but I don't know if we would have reason to support nonsmu devices without these three modes, or nonsmu devices with more than these three modes per channel. Could just fold the sl_mode_info definitions into a consts header in libsmu?